libs/application/templates/accident-notification/src/lib/dataSchema.ts

```diff
@@ -88,6 +88,12 @@ export const AccidentNotificationSchema = z.object({
     email: z.string().optional(),
     phoneNumber: z.string().optional(),
   }),
+  locationAndPurpose: z.object({
+    location: z.string().nonempty(),
```

Comment:
Maybe we should start using something else than nonempty since it's deprecated.
![image](https://user-images.githubusercontent.com/39527334/126985725-1d2699d9-42aa-4faf-bf99-b7c31b9ee4e9.png)

Reply:
Fixed and replaced all usage of nonempty on accident notification